packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/xcpd-request.ts

```diff
@@ -98,6 +100,12 @@ export async function processInboundXcpdRequest(
         jsonObj
       )}, request: ${request}`
     );
+    capture.error(msg, {
+      extra: {
+        error,
+        request,
+      },
+    });
```

Comment:
I don't think we should capture inbound errors. I understand we want to know about these so we can address them at some point, but inbound results in a lot of requests and some/most might be related to errors from external GWs.

We discussed this a few times, resulting in https://github.com/metriport/metriport-internal/issues/2425 and https://github.com/metriport/metriport-internal/issues/2378